### PR TITLE
Adding parser for IOSXE show_wireless_mobility_ap-list

### LINKED ISSUE #56: 31. show wireless mobility ap-list
```
ewlc1# show wireless mobility ap-list
AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-72-cap30                    58bf.eab3.1420    10.10.7.177      Self            
b80-81-cap4                     58bf.ea13.62a0    10.10.7.177      Self            
b80-52-cap6                     58bf.ea13.75e0    10.10.7.177      Self            
b80-61-cap14                    58bf.ea13.9100    10.10.7.177      Self            
b80-22-cap10                    58bf.ea37.0820    10.10.7.177      Self            
b80-71-cap1                     58bf.ea37.10a0    10.10.7.177      Self            
b80-61-cap7                     58bf.ea37.11c0    10.10.7.177      Self            
b80-52-cap18                    58bf.ea37.1d60    10.10.7.177      Self            
b80-71-cap3                     58bf.ea37.2ca0    10.10.7.177      Self            
b80-62-cap2                     58bf.ea37.34e0    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-01-cap16                    58bf.ea37.36c0    10.10.7.177      Self            
b80-32-cap3                     58bf.ea37.3960    10.10.7.177      Self            
b80-02-cap4                     58bf.ea37.40a0    10.10.7.177      Self            
b80-62-cap5                     58bf.ea37.4440    10.10.7.177      Self            
b80-51-cap9                     58bf.ea37.44c0    10.10.7.177      Self            
b80-02-cap8                     58bf.ea37.45c0    10.10.7.177      Self            
b80-22-cap14                    58bf.ea37.54a0    10.10.7.177      Self            
b80-72-cap8                     58bf.ea37.66a0    10.10.7.177      Self            
b80-72-cap6                     58bf.ea37.6dc0    10.10.7.177      Self            
b80-62-cap18                    58bf.ea37.75c0    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-22-cap16                    58bf.ea37.9e80    10.10.7.177      Self            
b80-31-cap17                    58bf.ea37.a520    10.10.7.177      Self            
b80-21-cap12                    58bf.ea37.afa0    10.10.7.177      Self            
b80-11-cap6                     58bf.ea37.c0c0    10.10.7.177      Self            
b80-82-cap1                     58bf.ea37.c7a0    10.10.7.177      Self            
b80-41-cap3                     58bf.ea37.c880    10.10.7.177      Self            
b80-81-cap14                    58bf.ea3c.c1c0    10.10.7.177      Self            
b80-62-cap11                    58bf.ea3c.e1a0    10.10.7.177      Self            
b80-61-cap11                    58bf.ea3c.f100    10.10.7.177      Self            
b80-81-cap6                     58bf.ea3d.2f20    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-31-cap11                    58bf.ea3d.3320    10.10.7.177      Self            
b80-52-cap10                    58bf.ea3d.3340    10.10.7.177      Self            
b80-61-cap12                    58bf.ea3d.35e0    10.10.7.177      Self            
b80-82-cap8                     58bf.ea3d.48c0    10.10.7.177      Self            
b80-61-cap6                     58bf.ea3d.5720    10.10.7.177      Self            
b80-31-cap12                    58bf.ea3d.7f80    10.10.7.177      Self            
b80-52-cap7                     58bf.ea3d.b640    10.10.7.177      Self            
b80-71-cap7                     58bf.ea3e.1be0    10.10.7.177      Self            
b80-62-cap4                     58bf.ea3e.1d60    10.10.7.177      Self            
b80-41-cap4                     58bf.ea3e.1da0    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-32-cap13                    58bf.ea3e.1e00    10.10.7.177      Self            
b80-01-cap11                    58bf.ea3e.1f60    10.10.7.177      Self            
b80-42-cap01                    58bf.ea3e.2020    10.10.7.177      Self            
b80-72-cap16                    58bf.ea3e.2040    10.10.7.177      Self            
b80-82-cap3                     58bf.ea3e.2060    10.10.7.177      Self            
b80-81-cap1                     58bf.ea3e.20c0    10.10.7.177      Self            
b80-22-cap2                     58bf.ea3e.20e0    10.10.7.177      Self            
b80-51-cap8                     58bf.ea3e.22a0    10.10.7.177      Self            
b80-52-cap3                     58bf.ea3e.22e0    10.10.7.177      Self            
b80-12-cap1                     58bf.ea3e.2420    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-32-cap11                    58bf.ea3e.24a0    10.10.7.177      Self            
b80-42-cap6                     58bf.ea3e.2500    10.10.7.177      Self            
b80-61-cap4                     58bf.ea3e.2540    10.10.7.177      Self            
b80-02-cap12                    58bf.ea3e.25a0    10.10.7.177      Self            
b80-11-cap1                     58bf.ea3e.2680    10.10.7.177      Self            
b80-52-cap13                    58bf.ea3e.2700    10.10.7.177      Self            
b80-71-cap5                     58bf.ea3e.2740    10.10.7.177      Self            
b80-11-cap3                     58bf.ea3e.2800    10.10.7.177      Self            
b80-72-cap13                    58bf.ea3e.2840    10.10.7.177      Self            
b80-31-cap7                     58bf.ea3e.2860    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-21-cap1                     58bf.ea3e.2880    10.10.7.177      Self            
b80-11-cap16                    58bf.ea3e.28a0    10.10.7.177      Self            
b80-82-cap15                    58bf.ea3e.2a60    10.10.7.177      Self            
b80-32-cap5                     58bf.ea3e.2a80    10.10.7.177      Self            
b80-32-cap6                     58bf.ea3e.2aa0    10.10.7.177      Self            
b80-71-cap2                     58bf.ea3e.2be0    10.10.7.177      Self            
b80-31-cap1                     58bf.ea3e.2c20    10.10.7.177      Self            
b80-31-cap9                     58bf.ea3e.2d00    10.10.7.177      Self            
b80-52-cap20                    58bf.ea3e.2f00    10.10.7.177      Self            
b80-31-cap15                    58bf.ea3e.2f20    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-62-cap17                    58bf.ea3e.2fa0    10.10.7.177      Self            
b80-12-cap10                    58bf.ea3e.2fe0    10.10.7.177      Self            
b80-41-cap14                    58bf.ea3e.3020    10.10.7.177      Self            
b80-02-cap7                     58bf.ea3e.3040    10.10.7.177      Self            
b80-32-cap4                     58bf.ea3e.3440    10.10.7.177      Self            
b80-61-cap19                    58bf.ea3e.35a0    10.10.7.177      Self            
b80-71-cap12                    58bf.ea3e.3660    10.10.7.177      Self            
b80-12-cap11                    58bf.ea3e.3680    10.10.7.177      Self            
b80-21-cap5                     58bf.ea3e.36e0    10.10.7.177      Self            
b80-41-cap15                    58bf.ea3e.3700    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-41-cap13                    58bf.ea3e.3740    10.10.7.177      Self            
b80-32-cap16                    58bf.ea3e.37a0    10.10.7.177      Self            
b80-12-cap3                     58bf.ea3e.3800    10.10.7.177      Self            
b80-11-cap10                    58bf.ea3e.3ba0    10.10.7.177      Self            
b80-42-cap2                     58bf.ea3e.3d00    10.10.7.177      Self            
b80-41-cap11                    58bf.ea3e.3d20    10.10.7.177      Self            
b80-01-cap15                    58bf.ea3e.4060    10.10.7.177      Self            
b80-22-cap6                     58bf.ea3e.41a0    10.10.7.177      Self            
b80-12-cap6                     58bf.ea3e.41c0    10.10.7.177      Self            
b80-61-cap18                    58bf.ea3e.41e0    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-52-cap4                     58bf.ea3e.4220    10.10.7.177      Self            
b80-51-cap1                     58bf.ea3e.4240    10.10.7.177      Self            
b80-72-cap1                     58bf.ea3e.4260    10.10.7.177      Self            
b80-12-cap7                     58bf.ea3e.4280    10.10.7.177      Self            
b80-71-cap16                    58bf.ea3e.42a0    10.10.7.177      Self            
b80-52-cap5                     58bf.ea3e.42c0    10.10.7.177      Self            
b80-31-cap5                     58bf.ea3e.4340    10.10.7.177      Self            
b80-01-cap2                     58bf.ea3e.43e0    10.10.7.177      Self            
b80-61-cap15                    58bf.ea3e.4400    10.10.7.177      Self            
b80-31-cap4                     58bf.ea3e.4420    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-01-cap14                    58bf.ea3e.4440    10.10.7.177      Self            
b80-32-cap9                     58bf.ea3e.4460    10.10.7.177      Self            
b80-02-cap13                    58bf.ea3e.44c0    10.10.7.177      Self            
b80-61-cap2                     58bf.ea3e.44e0    10.10.7.177      Self            
b80-62-cap8                     58bf.ea3e.4500    10.10.7.177      Self            
b80-01-cap7                     58bf.ea3e.4540    10.10.7.177      Self            
b80-01-cap13                    58bf.ea3e.4ae0    10.10.7.177      Self            
b80-21-cap9                     58bf.ea3e.4b20    10.10.7.177      Self            
b80-82-cap14                    58bf.ea3e.4bc0    10.10.7.177      Self            
b80-82-cap4                     58bf.ea3e.4be0    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-11-cap4                     58bf.ea3e.4cc0    10.10.7.177      Self            
b80-01-cap5                     58bf.ea3e.4ce0    10.10.7.177      Self            
b80-72-cap15                    58bf.ea3e.4d00    10.10.7.177      Self            
b80-01-cap6                     58bf.ea3e.4d40    10.10.7.177      Self            
b80-32-cap15                    58bf.ea3e.4d80    10.10.7.177      Self            
b80-42-cap7                     58bf.ea3e.4da0    10.10.7.177      Self            
b80-31-cap2                     58bf.ea3e.4e00    10.10.7.177      Self            
b80-32-cap8                     58bf.ea3e.4e20    10.10.7.177      Self            
b80-32-cap14                    58bf.ea3e.5480    10.10.7.177      Self            
b80-52-cap11                    58bf.ea3e.b5e0    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-61-cap9                     58bf.ea4e.f280    10.10.7.177      Self            
b80-81-cap9                     58bf.ea4e.f8e0    10.10.7.177      Self            
b80-72-cap7                     58bf.ea4f.0e80    10.10.7.177      Self            
b80-02-cap1                     58bf.ea4f.0f40    10.10.7.177      Self            
b80-21-cap7                     58bf.ea4f.4b80    10.10.7.177      Self            
b80-32-cap12                    58bf.ea84.cda0    10.10.7.177      Self            
b80-62-cap20                    58bf.ea84.d040    10.10.7.177      Self            
b80-72-cap4                     58bf.ea84.f100    10.10.7.177      Self            
b80-61-cap21                    58bf.ea84.f120    10.10.7.177      Self            
b80-82-cap13                    58bf.ea84.f140    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-32-cap2                     58bf.ea85.1060    10.10.7.177      Self            
b80-21-cap18                    58bf.ea85.2640    10.10.7.177      Self            
b80-11-cap8                     58bf.ea85.2700    10.10.7.177      Self            
b80-02-cap3                     58bf.ea85.29e0    10.10.7.177      Self            
b80-41-cap2                     58bf.ea85.4540    10.10.7.177      Self            
b80-42-cap16                    58bf.ea85.4800    10.10.7.177      Self            
b80-62-cap7                     58bf.ea85.4b20    10.10.7.177      Self            
b80-22-cap9                     58bf.ea85.5280    10.10.7.177      Self            
b80-02-cap6                     58bf.ea85.52a0    10.10.7.177      Self            
b80-42-cap5                     58bf.ea85.5320    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-71-cap4                     58bf.ea85.5340    10.10.7.177      Self            
b80-62-cap12                    58bf.ea85.5380    10.10.7.177      Self            
b80-81-cap10                    58bf.ea85.53e0    10.10.7.177      Self            
b80-52-cap9                     58bf.ea85.5400    10.10.7.177      Self            
b80-81-cap11                    58bf.ea85.5420    10.10.7.177      Self            
b80-61-cap1                     58bf.ea85.5540    10.10.7.177      Self            
b80-62-cap13                    58bf.ea85.55a0    10.10.7.177      Self            
b80-61-cap17                    58bf.ea85.5740    10.10.7.177      Self            
b80-01-cap12                    58bf.ea85.5780    10.10.7.177      Self            
b80-81-cap13                    58bf.ea85.5860    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-82-cap2                     58bf.ea85.5880    10.10.7.177      Self            
b80-52-cap16                    58bf.ea85.58e0    10.10.7.177      Self            
b80-82-cap7                     58bf.ea85.5940    10.10.7.177      Self            
b80-02-cap11                    58bf.ea85.5980    10.10.7.177      Self            
b80-21-cap4                     58bf.ea85.59a0    10.10.7.177      Self            
b80-51-cap4                     58bf.ea85.5ac0    10.10.7.177      Self            
b80-52-cap17                    58bf.ea85.5ae0    10.10.7.177      Self            
b80-21-cap14                    58bf.ea85.5b00    10.10.7.177      Self            
b80-11-cap7                     58bf.ea85.5b20    10.10.7.177      Self            
b80-72-cap17                    58bf.ea85.5b80    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-42-cap12                    58bf.ea85.5ba0    10.10.7.177      Self            
b80-51-cap12                    58bf.ea85.5c00    10.10.7.177      Self            
b80-51-cap10                    58bf.ea85.5c20    10.10.7.177      Self            
b80-22-cap7                     58bf.ea85.5c40    10.10.7.177      Self            
b80-12-cap18                    58bf.ea85.5ca0    10.10.7.177      Self            
b80-02-cap5                     58bf.ea85.5cc0    10.10.7.177      Self            
b80-02-cap9                     58bf.ea85.5ce0    10.10.7.177      Self            
b80-02-cap2                     58bf.ea85.5d00    10.10.7.177      Self            
b80-21-cap3                     58bf.ea85.5d80    10.10.7.177      Self            
b80-11-cap14                    58bf.ea85.5da0    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-11-cap15                    58bf.ea85.5de0    10.10.7.177      Self            
b80-72-cap9                     58bf.ea85.5ec0    10.10.7.177      Self            
b80-12-cap14                    58bf.ea85.5f40    10.10.7.177      Self            
b80-32-cap17                    58bf.ea85.6320    10.10.7.177      Self            
b80-61-cap8                     58bf.ea85.65e0    10.10.7.177      Self            
b80-31-cap14                    58bf.ea85.6620    10.10.7.177      Self            
b80-52-cap1                     58bf.ea85.6720    10.10.7.177      Self            
b80-31-cap8                     58bf.ea85.67e0    10.10.7.177      Self            
b80-41-cap1                     58bf.ea85.6880    10.10.7.177      Self            
b80-42-cap14                    58bf.ea85.68e0    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-41-cap5                     58bf.ea85.6920    10.10.7.177      Self            
b80-21-cap2                     58bf.ea85.6940    10.10.7.177      Self            
b80-12-cap2                     58bf.ea85.6960    10.10.7.177      Self            
b80-31-cap31                    58bf.ea85.69a0    10.10.7.177      Self            
b80-42-cap10                    58bf.ea85.69c0    10.10.7.177      Self            
b80-41-cap8                     58bf.ea85.69e0    10.10.7.177      Self            
b80-21-cap6                     58bf.ea85.6a00    10.10.7.177      Self            
b80-72-cap11                    58bf.ea85.6a20    10.10.7.177      Self            
b80-52-cap2                     58bf.ea85.6a40    10.10.7.177      Self            
b80-01-cap10                    58bf.ea85.6e20    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-12-cap9                     58bf.ea85.6fe0    10.10.7.177      Self            
b80-42-cap3                     58bf.ea85.7000    10.10.7.177      Self            
b80-51-cap5                     58bf.ea85.7020    10.10.7.177      Self            
b80-21-cap17                    58bf.ea85.7040    10.10.7.177      Self            
b80-21-cap13                    58bf.ea85.7080    10.10.7.177      Self            
b80-41-cap6                     58bf.ea85.7160    10.10.7.177      Self            
b80-61-cap16                    58bf.ea85.72c0    10.10.7.177      Self            
b80-41-cap10                    58bf.ea85.7300    10.10.7.177      Self            
b80-11-cap17                    58bf.ea85.7440    10.10.7.177      Self            
b80-01-cap4                     58bf.ea85.7520    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-51-cap15                    58bf.ea85.7600    10.10.7.177      Self            
b80-22-cap13                    58bf.ea85.7660    10.10.7.177      Self            
b80-61-cap5                     58bf.ea85.76a0    10.10.7.177      Self            
b80-62-cap9                     58bf.ea85.76e0    10.10.7.177      Self            
b80-52-cap21                    58bf.ea85.7700    10.10.7.177      Self            
b80-51-cap2                     58bf.ea85.7740    10.10.7.177      Self            
b80-62-cap10                    58bf.ea85.7760    10.10.7.177      Self            
b80-72-cap2                     58bf.ea85.77c0    10.10.7.177      Self            
b80-31-cap6                     58bf.ea85.7800    10.10.7.177      Self            
b80-51-cap3                     58bf.ea85.7860    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-72-cap18                    58bf.ea85.7cc0    10.10.7.177      Self            
b80-31-cap3                     58bf.ea85.7d00    10.10.7.177      Self            
b80-61-cap3                     58bf.ea85.7e40    10.10.7.177      Self            
b80-71-cap15                    58bf.ea85.7e60    10.10.7.177      Self            
b80-31-cap16                    58bf.ea85.7e80    10.10.7.177      Self            
b80-82-cap6                     58bf.ea85.7ea0    10.10.7.177      Self            
b80-11-cap9                     58bf.ea85.7f00    10.10.7.177      Self            
b80-42-cap9                     58bf.ea85.7f20    10.10.7.177      Self            
b80-22-cap4                     58bf.ea85.7f80    10.10.7.177      Self            
b80-82-cap12                    58bf.ea85.7fa0    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-82-cap10                    58bf.ea85.7fe0    10.10.7.177      Self            
b80-42-cap8                     58bf.ea85.8000    10.10.7.177      Self            
b80-12-cap16                    58bf.ea85.8060    10.10.7.177      Self            
b80-82-cap11                    58bf.ea85.80c0    10.10.7.177      Self            
b80-21-cap15                    58bf.ea85.80e0    10.10.7.177      Self            
b80-22-cap12                    58bf.ea85.8260    10.10.7.177      Self            
b80-62-cap22                    58bf.ea8d.a480    10.10.7.177      Self            
b80-52-cap19                    58bf.ea8d.a780    10.10.7.177      Self            
b80-72-cap14                    58bf.ea8d.aca0    10.10.7.177      Self            
b80-41-cap7                     58bf.ea8d.ace0    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-21-cap11                    58bf.ea8d.ad60    10.10.7.177      Self            
b80-82-cap9                     58bf.ea8d.ae40    10.10.7.177      Self            
b80-81-cap15                    58bf.ea8d.aec0    10.10.7.177      Self            
b80-41-cap12                    58bf.ea8d.afa0    10.10.7.177      Self            
b80-41-cap9                     58bf.ea8d.b040    10.10.7.177      Self            
b80-62-cap16                    58bf.ea8d.b120    10.10.7.177      Self            
b80-51-cap11                    58bf.ea8d.b140    10.10.7.177      Self            
b80-31-cap13                    58bf.ea8d.b1a0    10.10.7.177      Self            
b80-62-cap6                     58bf.ea8d.b220    10.10.7.177      Self            
b80-12-cap13                    58bf.ea8d.b260    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-81-cap5                     58bf.ea8d.b380    10.10.7.177      Self            
b80-01-cap9                     58bf.ea8d.b3e0    10.10.7.177      Self            
b80-52-cap15                    58bf.ea8d.b440    10.10.7.177      Self            
b80-42-cap15                    58bf.ea8d.b580    10.10.7.177      Self            
b80-62-cap14                    58bf.ea8d.b5a0    10.10.7.177      Self            
b80-12-cap15                    58bf.ea8d.b620    10.10.7.177      Self            
b80-42-cap13                    58bf.ea8d.b6c0    10.10.7.177      Self            
b80-72-cap12                    58bf.ea8d.b720    10.10.7.177      Self            
b80-61-cap10                    58bf.ea8d.b840    10.10.7.177      Self            
b80-62-cap21                    58bf.ea8d.b8e0    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-62-cap23                    58bf.ea8d.b900    10.10.7.177      Self            
b80-81-cap3                     58bf.ea8d.b940    10.10.7.177      Self            
b80-81-cap7                     58bf.ea8d.b9e0    10.10.7.177      Self            
b80-12-cap17                    58bf.ea8d.ba40    10.10.7.177      Self            
b80-31-cap10                    58bf.ea8d.ba80    10.10.7.177      Self            
b80-01-cap1                     58bf.ea8d.bbe0    10.10.7.177      Self            
b80-31-cap18                    58bf.ea8d.be00    10.10.7.177      Self            
b80-32-cap7                     58bf.ea8d.bf00    10.10.7.177      Self            
b80-82-cap17                    58bf.ea8d.bf60    10.10.7.177      Self            
b80-62-cap1                     58bf.ea8d.bf80    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-62-cap19                    58bf.ea8d.bfa0    10.10.7.177      Self            
b80-22-cap5                     58bf.ea8d.bfc0    10.10.7.177      Self            
b80-71-cap14                    58bf.ea8d.bfe0    10.10.7.177      Self            
b80-22-cap11                    58bf.ea8d.c020    10.10.7.177      Self            
b80-12-cap8                     58bf.ea8d.c040    10.10.7.177      Self            
b80-22-cap1                     58bf.ea8d.c060    10.10.7.177      Self            
b80-82-cap18                    58bf.ea8d.c080    10.10.7.177      Self            
b80-72-cap3                     58bf.ea8d.c1a0    10.10.7.177      Self            
b80-11-cap11                    58bf.ea8d.c260    10.10.7.177      Self            
b80-82-cap16                    58bf.ea8d.c2a0    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-81-cap12                    58bf.ea8d.c2c0    10.10.7.177      Self            
b80-01-cap8                     58bf.ea8d.c2e0    10.10.7.177      Self            
b80-81-cap2                     58bf.ea8d.c300    10.10.7.177      Self            
b80-51-cap14                    58bf.ea8d.c320    10.10.7.177      Self            
b80-21-cap8                     58bf.ea8d.c340    10.10.7.177      Self            
b80-11-cap13                    58bf.ea8d.c440    10.10.7.177      Self            
b80-61-cap13                    58bf.ea8d.c460    10.10.7.177      Self            
b80-51-cap13                    58bf.ea60.2b80    10.10.7.177      Self            
b80-12-cap4                     58bf.ea87.0240    10.10.7.177      Self            
b80-12-cap5                     58bf.ea87.4e80    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-71-cap6                     58bf.ea87.6220    10.10.7.177      Self            
b80-42-cap4                     58bf.ea9e.cc20    10.10.7.177      Self            
b80-81-cap8                     58bf.eaa1.4b80    10.10.7.177      Self            
b80-21-cap16                    58bf.eaa1.4bc0    10.10.7.177      Self            
b80-62-cap15                    58bf.eaa1.a8c0    10.10.7.177      Self            
b80-22-cap17                    58bf.eaa1.c2c0    10.10.7.177      Self            
b80-11-cap12                    58bf.eaa1.c580    10.10.7.177      Self            
b80-32-cap10                    58bf.eaa2.2840    10.10.7.177      Self            
b80-22-cap3                     58bf.eaa2.58c0    10.10.7.177      Self            
b80-01-cap3                     58bf.eaa2.6320    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-52-cap12                    58bf.eaa2.7d20    10.10.7.177      Self            
b80-71-cap11                    58bf.eaa2.81c0    10.10.7.177      Self            
b80-11-cap2                     58bf.eaa2.8c00    10.10.7.177      Self            
b80-11-cap5                     58bf.eaa2.9340    10.10.7.177      Self            
b80-32-cap1                     58bf.eaa2.96c0    10.10.7.177      Self            
b80-22-cap8                     58bf.eabe.8ae0    10.10.7.177      Self            
b80-61-cap20                    58bf.eace.ae00    10.10.7.177      Self            
b80-71-cap13                    58bf.eace.afc0    10.10.7.177      Self            
b80-02-cap10                    58bf.eace.ce20    10.10.7.177      Self            
b80-71-cap10                    58bf.eacf.1de0    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-42-cap11                    58bf.eacf.a520    10.10.7.177      Self            
b80-21-cap10                    58bf.eacf.b200    10.10.7.177      Self            
b80-71-cap9                     58bf.eacf.c540    10.10.7.177      Self            
b80-52-cap14                    58bf.eacf.c6e0    10.10.7.177      Self            
b80-72-cap10                    58bf.eacf.db00    10.10.7.177      Self            
b80-71-cap8                     58bf.eacf.dce0    10.10.7.177      Self            
b80-62-cap3                     58bf.eacf.e520    10.10.7.177      Self            
b80-51-cap7                     58bf.eacf.e5c0    10.10.7.177      Self            
b80-52-cap8                     58bf.ead0.3700    10.10.7.177      Self            
b80-12-cap12                    58bf.ead0.38c0    10.10.7.177      Self            

AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-51-cap6                     58bf.eafe.8e40    10.10.7.177      Self            

```

### PR DESCRIPTION
**Pull request by robschn**

### CODE REVIEW

File: src/genie/libs/parser/iosxe/tests/ShowWirelessMobilityApList/cli/equal/golden_output_expected.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+expected_output = {
```
Comment (itdependsnetworks) [inline src/genie/libs/parser/iosxe/tests/ShowWirelessMobilityApList/cli/equal/golden_output_expected.py:1]: Do you mind running black on this?

File: src/genie/libs/parser/iosxe/show_wireless.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    """Parser for show wireless mobility ap-list"""
+
+    cli_command = ['show wireless mobility ap-list']
+
+    def cli(self, output=None):
+        if output is None:
+            output = self.device.execute(self.cli_command[0])
+        else:
+            output = output
+
```
Comment (itdependsnetworks) [inline src/genie/libs/parser/iosxe/show_wireless.py:198]: Can you put in a small (but complete) example in a comment here

File: src/genie/libs/parser/iosxe/show_wireless.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        ap_info_capture = re.compile(
+                    r"^(?P<ap_name>\S+)\s+(?P<radio_mac>\S+)\s+(?P<controller_ip>\d+\.\d+\.\d+\.\d+)\s+(?P<learnt_from>\S+)") # could also do (?P<learnt_from>(Self))
+
+        ap_info_obj = {}
+
+        for line in output.splitlines():
+
+            if ap_header_capture.match(line):
+                continue
+            if delimiter_capture.match(line):
```
Comment (itdependsnetworks) [inline src/genie/libs/parser/iosxe/show_wireless.py]: can make elif

File: src/genie/libs/parser/iosxe/show_wireless.py (hunk 2 above)
Comment (itdependsnetworks) [reply src/genie/libs/parser/iosxe/show_wireless.py:198]: like this size 
```
AP name                           AP radio MAC      Controller IP     Learnt from
--------------------------------------------------------------------------------------
b80-72-cap30                    58bf.eab3.1420    10.10.7.177      Self            
b80-81-cap4                     58bf.ea13.62a0    10.10.7.177      Self            
b80-52-cap6                     58bf.ea13.75e0    10.10.7.177      Self    
```

File: src/genie/libs/parser/iosxe/tests/ShowWirelessMobilityApList/cli/equal/golden_output_expected.py (hunk 1 above)
Comment (robschn) [reply src/genie/libs/parser/iosxe/tests/ShowWirelessMobilityApList/cli/equal/golden_output_expected.py:1]: Done!

File: src/genie/libs/parser/iosxe/show_wireless.py (hunk 3 above)
Comment (robschn) [reply src/genie/libs/parser/iosxe/show_wireless.py]: changed to elif

**Comment by itdependsnetworks** [comment]
Please submit PR to Cisco and note the request here.

**Comment by robschn** [comment]
Pull request created to Cisco!

https://github.com/CiscoTestAutomation/genieparser/pull/243

**Pull request merged**